content/en/agent/faq/ec2-use-win-prefix-detection.md

```diff
@@ -7,7 +7,7 @@ aliases:
 
 ## Issue
 
-In the Datadog Agent v6 and v7, the default Agent in-app hostname for Unix platforms on EC2 is the instance-id. 
+In the Datadog Agent v6 & v7, the default Agent in-app hostname for Unix platforms on EC2 is the instance-id. 
```

Comment:
I'm surprised at the use of the ampersand outside of a tab heading, but it seems intentional, so I'm assuming there's a style rule I don't know, and leaving it :)

Note that some instances of "v6 and v7" still appear in some spots in this PR, I'm not sure whether you want to replace those as well?

Reply:
Good catch. I was originally only trying to change the tab headings, cos they were inconsistent. But I caught too many of them in my find/replace.